extra/1-radio-stations.js

```diff
@@ -26,6 +31,17 @@
  */
 // `getStations` goes here
 
+function getStations(){
+   let radio = getAllFrequencies();
+   let storage = [];
+   for (let char of radio) {
+    if (isRadioStation(char)) {
+       storage.push(char);
+    } 
+   }
+
+return storage;
+}
```

Comment:
```suggestion
function getStations() {
  const allFrequencies = getAllFrequencies();
  const radioFrequencies = [];

  for (let i = 0; i < allFrequencies.length; i++) {
    const currentFrequency = allFrequencies[i];
    if (isRadioStation(currentFrequency))
      radioFrequencies.push(currentFrequency);
  }

  return radioFrequencies;
}
```
Good job for writing the function! 
The variable names are not intuitive and could be confusing for other people reading the code. Check out the suggestion for better variable names.